Configure

```diff
@@ -360,6 +360,8 @@ $config{release_date} = $version{RELEASE_DATE} // 'xx XXX xxxx';
 
 $config{version} = "$config{major}.$config{minor}.$config{patch}";
 $config{full_version} = "$config{version}$config{prerelease}$config{build_metadata}";
+$config{FIPSVENDOR} =
+    (defined $version{FIPSVENDOR} ? "$version{FIPSVENDOR} " : "") . "OpenSSL FIPS Provider";
```

Comment:
The exact form of this wording needs internal discussion. That might take a while.

Reply:
@mattcaswell I am assuming comment with a new name by @nhorman is authoritative. Please re-review.